src/cs/Ssh/SecureStream.cs

```diff
@@ -0,0 +1,291 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.DevTunnels.Ssh.Events;
+
+namespace Microsoft.DevTunnels.Ssh;
+
+/// <summary>
+/// Establishes an end-to-end encrypted two-way authenticated data stream over an underlying
+/// transport stream, using the SSH protocol but providing simplified interface that is limited to
+/// a single duplex stream (channel).
+/// </summary>
+/// <remarks>
+/// This class is a complement to <see cref="MultiChannelStream"/>, which provides only the
+/// channel-multiplexing functions of SSH.
+/// 
+/// To establish a secure connection, the two sides first establish an insecure transport stream
+/// over a pipe, socket, or anything else. Then they encrypt and authenticate the connection
+/// before beginning to send and receive data.
+/// </remarks>
+public class SecureStream : Stream
```

Comment:
Does this class need `channelMaxWindowSize` for parity with ts?

Reply:
Oh, that TypeScript `SecureStream.channelMaxWindowSize` property is not used, It was a mistake from copy-pasting code from `MultiChannelStream`. I'm deleting it from `SecureStream`.